### ISSUE TITLE
Pytorch Preprocessing Mismatch

### ISSUE THREAD

**Issue by Wei-HaiMing** [Strawberry64/Pareidolia-Desktop#65]
Issuer: Armando Vega

 Date: 04-24-2026

 Operating System: Windows & MacOS

 Branch Tested On: tensorUpdate

 Description: The normalization for Pytorch is happening twice from the DataLoader objects, the preprocessing wrapper, and the representative dataset. Standardize the normalization to the DataLoader level. 

 Replication Steps: N/A, just look at the code.
